src/GeositeFramework/js/SinglePluginModeHelp.js

```diff
@@ -2,9 +2,13 @@
     'use strict';
 
     var $container;
+    var $activePlugin;
+    var $pluginToggle;
 
     function initialize(view, $el) {
         $container = $el;
+        $activePlugin = $('.sidebar:visible');
```

Comment:
This isn't actually selecting the plugin on `initialize`, doesn't look like it's in the DOM yet.  This plugin isn't actually hidden as a result.

Reply:
Weird. I must have screwed something up before checking it in. It was hiding for me initially. Sorry. I'll take another look.